src/android/CodePushPackageManager.java

```diff
@@ -95,4 +95,16 @@ public InstallOptions getPendingInstall() {
     public void clearPendingInstall() {
         this.codePushPreferences.clearPendingInstall();
     }
+
+    public void markUnconfirmedInstall(boolean unconfirmed) {
```

Comment:
I agree with @geof90 the name is confusing a bit, if you want to confirm install, we pass markUnconfirmedInstall(false) - double negation.. atleast for me makes it hard to read.

Having two methods would be a lot clearer or may be changing this to ```markInstall(boolean isConfirmed)``` that way when I say markInstall(true) this is a confirmed installl markInstall(false) is a an unconfirmed. If that makes sense. Just a suggestion.